tests/conftest.py

```diff
@@ -129,6 +129,30 @@ def _test_images_s3_bucket():
 MICROVM_S3_FETCHER = MicrovmImageS3Fetcher(_test_images_s3_bucket())
```

Comment:
the commit message should be modified. it adds the `ExceptionAggregator` abstraction, not a utility function